tools/src/main/python/dlpx/virtualization/_internal/cli.py

```diff
@@ -32,6 +32,12 @@
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'],
                         obj=click_util.ConfigFileProcessor.read_config())
 
+# This setting is added to workaround the bug in click 7.1 on windows
+# when case_sensitive=False is used on click.Options
+CONTEXT_SETTINGS_INIT = dict(help_option_names=['-h', '--help'],
+                             obj=click_util.ConfigFileProcessor.read_config(),
+                             token_normalize_func=lambda x: x.encode("ascii"))
```

Comment:
are we sure we want to only change the init command to not be case sensitive or does it make sense to set the whole click group to have this setting? ie what if we just update context_setting? Or was tehre a specific reason you didn't just do that and if so why is it only init that we want to change? I think just thinking about it high level, all the cli commands i feel like should do the same thing?

Reply:
case_sensitive=False is used only in click.Options of init command. I chose to separate it out into a new CONTEXT_SETTINGS_INIT instead of adding to existing CONTEXT_SETTINGS to reduce the side effects of it (if any) on other commands.